src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/HeaderSWA.tsx

```diff
@@ -43,9 +46,13 @@ export const HeaderSWA = () => {
         </Text>
 
         <Media greaterThanOrEqual="sm">
-          <Spacer y={[2, 4]} />
+          <Spacer y={[2, 1]} />
 
           <GridColumns>
+            <Column span={[12, 12, 10]}>
+              {enableNewSubmissionFlow && <PreviousSubmissionQueryRenderer />}
+            </Column>
+
```

Comment:
```
...
<Spacer y={[2, 4]} />

{enableNewSubmissionFlow && 
 <Column span={[12, 12, 10]}>
       <Spacer y={[2, 1]} />
       <PreviousSubmissionQueryRenderer />
 </Column>}
...
```
Otherwise, the spacer breaks if the ff is disabled